custom/v3/note/data/src/main/java/org/apache/fineract/v3/note/data/NoteResponseData.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.v3.note.data;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.Map;
+import lombok.Data;
+import lombok.experimental.FieldNameConstants;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@FieldNameConstants
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Let's define these behaviors of the JSON parser when we configure Jackson on a global level... otherwise you'll have to be consistent and use that annotation everywhere... let's avoid boilerplate code where we can (read: other people will forget these things very easily and then the code quality suffers again).

Reply:
It's already configured globally in [`application-v3.properties`](https://github.com/Zeyad2003/fineract/blob/18c2d660373657fa7a55a652d9088eaeae75c45c/custom/v3/common/configuration/src/main/resources/application-v3.properties#L29)
Forgot to remove it from other data classes. 
Removed now.